.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build
+on: [push, pull_request]
+
```

Comment:
You should add branch name, where to want to build

Reply:
Ok. I've added branch name. Let me know if any changes are required in this.